test/SiftClientTest.php

```diff
@@ -658,10 +658,41 @@ public function testApplyDecisionToContentWithSpecialCharacters() {
     public function testTrackProfileEvent() {
         $mockUrl = 'https://api.sift.com/v205/events';
         $mockResponse = new SiftResponse('{"status": 0, "error_message": "OK"}', 200, null);
-        SiftRequest::setMockResponse($mockUrl, SiftRequest::POST ,$mockResponse);
+        SiftRequest::setMockResponse($mockUrl, SiftRequest::POST, $mockResponse);
 
         $response = $this->client->track('$create_content', $this->profile_properties);
         $this->assertTrue($response->isOk());
         $this->assertEquals($response->apiErrorMessage, 'OK');
     }
+
+    public function testSend() {
+        $mockUrl = 'https://api.sift.com/v1.1/verification/send';
```

Comment:
Did these tests pass with the version being wrong in the actual send function? I think it should have caught the bug in the test cases when you ran them.

Reply:
Yes..It was my mistake. sorry!